src/components/DefineCompany/BusinessContactInfo.vue

```diff
@@ -10,9 +10,10 @@
         </v-flex>
         <v-flex md4>
           <div><label><strong>Phone Number</strong></label></div>
-          <div id="lbl-phone">{{ !!contact.phone ? contact.phone : 'Not entered' }}
-            Ext: {{ !!contact.extension ? contact.extension : 'Not entered' }}
+          <div id="lbl-phone" v-if="!!contact.phone">{{ contact.phone }}
+            <span v-if="!!contact.extension">Ext: {{ contact.extension }}</span>
           </div>
+          <div id="lbl-phone" v-else>Not entered</div>
```

Comment:
Display is now consistent with other Contact Info fields. And it doesn't show the extension if none was entered.